### ISSUE TITLE
Change resource status to CURATED at the first Publish

### ISSUE THREAD

**Issue by lyndonnixon** [linkedtv/editortool#47]
@jthomsen "yes. the first "Publish" would be the best imho; Jaap could do this via a REST call, just as Pieter sets the status to TRANSFERRED as soon as it is transferred and encoded"

**Comment by jblom** [comment]
In this case I would simply set the status curated every time you publish. It should be a harmless call to the platform right?

**Comment by lyndonnixon** [comment]
yes although I would actually say only the FIRST time one publishes. seems to be a lot of unnecessary additional calls after that. I am not sure what Unpublish does (do they always (un)publish ALL the curated data or just the data in the active chapter? Since the options appear in the chapter section one assumes they apply only to that chapter) but it could then reset status to ANNOTATED if deleting ALL the curated data (althiugh that seems a bit dangerous since one could always accidentally hit it)

**Comment by jblom** [comment]
I really don't want to keep track of an extra thing if it's not necessary. It only adds another silly field to the ET database. Also publishing is not something that happens a lot. Only when you press that button, so it's exactly a wave of HTTP requests the platform has to endure. Also it's actually good to always set it to CURATED, since it might be possible that some other process sets the status on the platform to something else again. In those cases you would want the status to be always set to CURATED in any case.

Also the publish is for the entire mediaresource. The button is in a not so handy place, I admit. It should be moved somewhere at the top.

Also unpublish removes all the curated RDF triples from the platform. (so publish is only needed to add triples to the store and set the status to CURATED)

**Comment by lyndonnixon** [comment]
ok, agree: publish: change status to CURATED. unpublish: change status to ANNOTATED. 
in a later version of ET, yes, these options would better be placed in a "global" part of the ET UI, e.g, the video section. 
It would be nice also to be able to publish/unpublish only the annotations for a specific chapter (whixh is what I thought I was doing as i was annotating this week), since this is what probably curators would change most over time (given that enrichment links particularly can be very time dependent)
